File: frontend/apps/ios/Pantopus/Features/SupportTrains/StartTrain/StartSupportTrainSampleData.swift
//
//  StartSupportTrainSampleData.swift
//  Pantopus
//
//  Deterministic A12.11 data for previews, structural snapshots, and
//  step-1 invite/verified-recipient fixtures.
//

import Foundation

public enum StartSupportTrainSampleData {
    public static let verifiedNeighbor = MailRecipientDTO(
        userId: "u_maya_patel",
        name: "Maya Patel",
        username: "maya",
        homeId: "home_elm_418",
        homeAddress: "418 Elm St, Apt 2",
        isVerified: true,
        homeMediaUrl: nil,
        isOnPantopus: true
    )

    public static let verifiedContextNote =
        "Maya is home after knee surgery on the 12th. Meals, dog walks for Pixel, and rides to PT would help."

    public static let inviteQuery = "David Chen"
}

File: frontend/apps/android/app/src/main/java/app/pantopus/android/ui/screens/support_trains/start_train/StartSupportTrainSampleData.kt
@file:Suppress("PackageNaming")

package app.pantopus.android.ui.screens.support_trains.start_train

import app.pantopus.android.data.api.models.mail_compose.MailRecipientDto

/** Deterministic A12.11 fixtures for previews and snapshots. */
object StartSupportTrainSampleData {
    val verifiedNeighbor =
        MailRecipientDto(
            userId = "u_maya_patel",
            name = "Maya Patel",
            username = "maya",
            homeId = "home_elm_418",
            homeAddress = "418 Elm St, Apt 2",
            isVerified = true,
            homeMediaUrl = null,
            isOnPantopus = true,
        )

    const val VERIFIED_CONTEXT_NOTE: String =
        "Maya is home after knee surgery on the 12th. Meals, dog walks for Pixel, and rides to PT would help."

    const val INVITE_QUERY: String = "David Chen"
}
